pkg/credentials/noop.go

```diff
@@ -0,0 +1,19 @@
+package credentials
+
+type NoopStore struct{}
+
+func (s NoopStore) Get(_ string) (*Credential, bool, error) {
```

Comment:
nit, here and below:
```suggestion
func (NoopStore) Get(string) (*Credential, bool, error) {
```